debug/mutex_test.go

```diff
@@ -65,21 +65,20 @@ func TestRWMutexReportEvery(t *testing.T) {
 	}()
 
 	// Wait for writer to appear
-	for {
-		if atomic.LoadInt64(&mutex.writers) != 1 {
-			time.Sleep(time.Millisecond)
-		}
-		break
+	for atomic.LoadInt64(&mutex.writers) != 1 {
```

Comment:
Confirming that you intended to change semantics ~ the original code sleeps at most once, and then breaks out the loop. The new code is going to sleep repeatedly till the condition is true.